compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/SchemaGenerationTest.java

```diff
@@ -204,92 +193,6 @@ public void testGraphqlServiceWithHttpService() {
         Assert.assertEquals(diagnosticResult.errorCount(), 0);
     }
 
-    @Test(enabled = false)
```

Comment:
Since we are removing the schema string generation tests here, shouldn't we at least have unit tests for those APIs?

Reply:
Since these tests will be added back to the GraphQL CLI tool, this will be tested there. Since we haven't used this in GraphQL Package, shall we add unit tests later on via separate PR? Seems it's a bit complex to add unit tests.